core-common/src/main/java/org/zalando/nakadi/domain/VersionedAvroSchema.java

```diff
@@ -18,4 +18,8 @@ public Schema getSchema() {
     public String getVersion() {
         return version;
     }
+
+    public byte getVersionAsByte() {
```

Comment:
I think version can be something different other than 1 byte. it is a string in nature.

Reply:
discard that, that is metadata version, which is part of the envelop